.github/workflows/release.yml

```diff
@@ -12,6 +12,9 @@ on:
       homebrew-prerelease:
         description: 'HomeBrew prerelease'
         default: 'false'
+      draft-release:
+        description: 'Set to true to do a draft release for sanity testing'
```

Comment:
Don't think you need to mention setting the flag here. You can move this to the doc

Reply:
That's right. I'll do that.